Description: Patch to increase the minimum size of the city dialogue
box so there is no scroll on the minimap (TODO: test this actually
works, especially on small screens)
Author: Karl Goetz <karl@kgoetz.id.au>
Forwarded: no
Last-Update: <2010-08-12>

Index: freeciv-2.4.0/client/options.h
===================================================================
--- freeciv-2.4.0.orig/client/options.h	2013-09-14 23:00:08.645475499 +0200
+++ freeciv-2.4.0/client/options.h	2013-09-14 23:00:08.637475499 +0200
@@ -367,7 +367,7 @@
 #define GUI_GTK2_CITYDLG_MIN_XSIZE      256
 #define GUI_GTK2_CITYDLG_MAX_XSIZE      4096
 
-#define GUI_GTK2_CITYDLG_DEFAULT_YSIZE  512
+#define GUI_GTK2_CITYDLG_DEFAULT_YSIZE  524
 #define GUI_GTK2_CITYDLG_MIN_YSIZE      128
 #define GUI_GTK2_CITYDLG_MAX_YSIZE      4096
 
